### PR TITLE
partial ui changes done

### PR DESCRIPTION
**Pull request by manoj-surabathulla**
- Created UI design with a mobile response.
- Created slice for the toggle.
- Cleaned up the functionality code and placed all functions in the service folder in the database.service file.

### CODE REVIEW

**Review by afrze — APPROVED** [review]
LGTM!

File: src/services/firebase/auth.service.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  try {
+    const res: any = await signInWithPopup(auth, provider);
+    const userDocRef = doc(db, "users", res.user.uid);
+    const userDocSnap = await getDoc(userDocRef);
+    const userResults = {
+      uid: res.user.uid,
+      email: res.user.email,
+      displayName: res.user.displayName,
+      acessToken: res?.user?.stsTokenManager?.accessToken,
```
Comment (Vivekflyerssoft) [inline src/services/firebase/auth.service.ts:29]: assign res?.user?.stsTokenManager? in a const an use it, avoid multiple ?

Comment (manoj-surabathulla) [reply src/services/firebase/auth.service.ts:29]: yeah okay bro

**Comment by Vivekflyerssoft** [comment]
@manoj-surabathulla  Take the latest, I have pushed in couple of commit :)

**Pull request merged**